app/components/map/index.js

```diff
@@ -521,14 +525,12 @@ Map.propTypes = {
     coordinates: React.PropTypes.array.isRequired
   }).isRequired,
   center: React.PropTypes.shape({
-    lat: React.PropTypes.number.isRequired,
-    lng: React.PropTypes.number.isRequired
+    lat: React.PropTypes.number,
```

Comment:
I think they should be required. I mean, the object itself is not mandatory but if we pass that object we should always pass the lat and lon values.
The warnings that we are getting is because the object that the container pass has `lon` instead of `lng` attribute so updating the name in the PropTypes would be enough to remove the warning.